Dockerfile

```diff
@@ -42,8 +43,8 @@ COPY --chown=${WSGI_USER}:${WSGI_USER} . .
 
 ENV VIRTUAL_ENV="/app/.venv"
 ENV PATH="${VIRTUAL_ENV}/bin:${PATH}"
-RUN mkdir static && python manage.py collectstatic
+RUN python manage.py collectstatic
 
 EXPOSE 8000
 ENV UWSGI_HTTP=:8000 UWSGI_MASTER=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_AUTO_CHUNKED=1 UWSGI_WSGI_ENV_BEHAVIOUR=holy
-CMD ["uwsgi", "-w", "recvalsite.wsgi", "--static-map", "/static=/var/www/recvalsite/static"]
+CMD ["uwsgi", "-w", "recvalsite.wsgi", "--processes", "10", "--static-map", "/static=/var/www/recvalsite/static"]
```

Comment:
I think there's also a place in the nginx file where we could make this change? It's probably fine here for now though

Reply:
itwewina is set up to use a [`uwsgi.ini`](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/master/CreeDictionary/uwsgi.ini) file for [almost all](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/master/docker/Dockerfile#L107) the uwsgi settings; this is the “let me try to get the site working as quickly as possible” version.

I’ll make a note to clean this up later.